language/tools/test-generation/src/lib.rs

```diff
@@ -128,7 +155,7 @@ pub fn run_generation(args: Args) {
             }
             Err(e) => {
                 error!("{}", e);
-                output_error_case(module.clone(), args.output_path.clone(), i);
+                output_error_case(module.clone(), args.output_path.clone(), i, seed);
```

Comment:
What is the purpose of saving the seed with each erroneous output?

Reply:
The idea being that we could use this seed to regenerate the module (the iteration and initial seed). I'm not super wedded to it, so happy to pull it out if you don't think it will be that useful.

Reply:
not sure I follow here. I thought the seed was key in getting a stable repro, is it?
if that is the case I would be happy to have it everywhere we can. If not, than it does not matter

Reply:
In terms of getting a repro, any module that may cause issues is saved. However, having a seed can be quite good for development of the tool (in order to make sure issues are addressed), as well as to repro without downloading the module. I'll take this out for now :)